bedrock/firefox/redirects.py

```diff
@@ -76,11 +21,22 @@ def firefox_channel(*args, **kwargs):
     return platform_redirector("firefox.channel.desktop", "firefox.channel.android", "firefox.channel.ios")
 
 
+def validate_param_value(param: str | None, default: str | None = None) -> str | None:
+    """
+    Returns the value passed in if it contains only letters, numbers, dashes, and underscores.
+    Otherwise returns `default`.
```

Comment:
I'd put the pattern comment with the regex. If that changes, this comment invalidates.